<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="searchGrid">
      <item row="0" column="2" colspan="2">
       <widget class="QPushButton" name="search_2">
        <property name="text">
         <string>Найти</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="replaceAll">
        <property name="text">
         <string>Заменить всё</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="replaceLabel">
        <property name="text">
         <string>Текст для замены</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="searchLabel">
        <property name="text">
         <string>Текст для поиска</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="searchLineEdit"/>
      </item>
      <item row="0" column="4" rowspan="3">
       <widget class="QPushButton" name="closeSearch">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Закрыть</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="replace_2">
        <property name="text">
         <string>Заменить</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="replaceLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="mainTextEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>955</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="create"/>
    <addaction name="open"/>
    <addaction name="save"/>
    <addaction name="save_as"/>
    <addaction name="separator"/>
    <addaction name="close"/>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Редактирование</string>
    </property>
    <addaction name="undo"/>
    <addaction name="search"/>
    <addaction name="replace"/>
    <addaction name="clear"/>
    <addaction name="copy"/>
    <addaction name="paste"/>
    <addaction name="fontEdit"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="create"/>
   <addaction name="open"/>
   <addaction name="save"/>
   <addaction name="save_as"/>
   <addaction name="close"/>
   <addaction name="undo"/>
   <addaction name="search"/>
   <addaction name="replace"/>
   <addaction name="clear"/>
   <addaction name="copy"/>
   <addaction name="paste"/>
   <addaction name="fontEdit"/>
   <addaction name="exit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="create">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/images/create.png</normaloff>:/icons/images/create.png</iconset>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
  </action>
  <action name="open">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/images/open.png</normaloff>:/icons/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Открыть...</string>
   </property>
  </action>
  <action name="save">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/images/save.png</normaloff>:/icons/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="save_as">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/images/saveAs.png</normaloff>:/icons/images/saveAs.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
  <action name="search">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/images/search.png</normaloff>:/icons/images/search.png</iconset>
   </property>
   <property name="text">
    <string>Поиск</string>
   </property>
  </action>
  <action name="replace">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/images/replace.png</normaloff>:/icons/images/replace.png</iconset>
   </property>
   <property name="text">
    <string>Замена</string>
   </property>
  </action>
  <action name="clear">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/images/clear.png</normaloff>:/icons/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
  </action>
  <action name="copy">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/images/copy.png</normaloff>:/icons/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>Копировать</string>
   </property>
  </action>
  <action name="paste">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/images/paste.png</normaloff>:/icons/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>Вставить</string>
   </property>
  </action>
  <action name="undo">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/images/undo.png</normaloff>:/icons/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>Отменить</string>
   </property>
  </action>
  <action name="exit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/images/exit.png</normaloff>:/icons/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="close">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/images/close.png</normaloff>:/icons/images/close.png</iconset>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </action>
  <action name="fontEdit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/images/fontStyle.png</normaloff>:/icons/images/fontStyle.png</iconset>
   </property>
   <property name="text">
    <string>Редактировать текст...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
